tests/Optimizer/Transformer/OptimizeHeroImagesTest.php

```diff
@@ -341,6 +341,28 @@ public function dataTransform()
                     . '</amp-story-player>'
                 ),
             ],
+
+            'preserves fetchpriority attribute from noscript fallback img' => [
+                $input(
+                    '<amp-img width="500" height="400" src="/img1.png"><noscript><img src="/img1.png" width="500" height="400" fetchpriority="high"></noscript></amp-img>'
+                ),
+                $output(
+                    '<amp-img data-hero width="500" height="400" src="/img1.png" i-amphtml-ssr><img class="i-amphtml-fill-content i-amphtml-replaced-content" decoding="async" fetchpriority="high" src="/img1.png"></amp-img>'
+                ),
+            ],
+
```

Comment:
Am I right that this test is somewhat redundant with the second?

Reply:
Yes.

Reply:
Updated the test cases in https://github.com/ampproject/amp-toolbox-php/pull/544/commits/16b9f6689b66031fd09d512955fb1bae36236115